src/components/dev-hub/global-nav.js

```diff
@@ -74,7 +74,7 @@ export default () => {
     return (
         <GlobalNav>
             <NavContent>
-                <HomeLink to="/">
+                <HomeLink aria-label="Home" to="/">
```

Comment:
Would title work here as well, or is aria-label the recommended approach?

Reply:
Specifically for screen-readers, reading a `title` [isn't always enabled by default](https://www.mediacurrent.com/blog/dont-rely-title-attribute-accessibility-seo/):

```
A person who uses a screen reader because they are blind or have low vision will likely never have the title attribute read to them. Some screen readers don’t support the title attribute and those that do, don’t read the title attribute by default. It requires changing a setting for the title to be read, which not all users do.
```

Are there other use cases for the `title` attribute?

Reply:
Wow, TIL. I was always under the impression title served two purposes
- add a hover tooltip
- add accessibility text

Reply:
`title` does add that tooltip, so it seems the most bulletproof way would be to have both an `aria-label` as well as a `title`, but it also seems redundant. Thoughts?

Reply:
I'm fine with just the label